lib/jnpr/junos/device.py

```diff
@@ -897,6 +907,7 @@ def __init__(self, *vargs, **kvargs):
             self._hostname = 'localhost'
             self._ssh_private_key_file = None
             self._ssh_config = None
+            self._ssh_keepalives = None
```

Comment:
Setting this to ``None`` does not make sense.  This value is initialized to zero in the underlying module. 

Help on method set_keepalive in module paramiko.transport:

set_keepalive(self, interval) method of paramiko.transport.Transport instance
    Turn on/off keepalive packets (default is off).  If this is set, after
    ``interval`` seconds without sending any data over the connection, a
    "keepalive" packet will be sent (and ignored by the remote host).  This
    can be useful to keep connections alive over a NAT, for example.
    
    :param int interval:
        seconds to wait before sending a keepalive packet (or
        0 to disable keepalives).

Reply:
None is set for on-box python, where it won't use ssh connection. So this should be OK